### ISSUE TITLE
Spymaster Mode

### ISSUE THREAD

**Issue by CMMRM63** [jbowens/codenames#124]
Hello

We discovered this life-saver-of-a-game during times of COVID-19. 
Playing with friends and family has brought much joy and togetherness in times of isolation. However, during these depressing and trying times, where people have low morale and feel crappy about themselves, there is clearly cheating happening with crazy super-power abilities emerging suddenly in the connection of words...Professor X would be so proud.

I kid you not, during one game, a guesser wrote down all the words on a piece of paper, and in an epic Piccasio-esque-venn diagram, showed their deduction process of picking the correct words...HMM CHEAT!

Having a restricted spymaster mode would bring much joy to my life as I am a stickler for rules and try to lead a fair and honest life. I just don't dig the cheating and sudden psychic abilities.

Is there a way of having a spymaster mode:
- allowing only two people to press spymaster
- having a password entered
- something else to STOP the cheating!

It would be the most amazing milestone in my life to find a solution to this...and I am sure other codenamers' lives!
I hope you hear my cry for help and are able to develop a way forward to #STOPTHECHEATING

Otherwise, I will need to find new friends...

**Comment by CWempe** [comment]
I don't think you can prevent someone from writing down the words on paper. 😄 

But I would support an anti-cheat solution.

I was thinking of something like this:

There is a game master who is kind of the administrator for the whole game.
The game master can set global settings like word set, timer, etc.
And he can create unique URLs for the Spamasters.
The default url would only show guess mode.

Maybe the spy masters could be the game master, too.
So we would not need a third role.

tl;dr:
**unique url instead of password**

But password would be fine, too.

**Comment by sekelsta** [comment]
I hope that if that happens, there'll still be a way to play normally. With my group, it's very convenient that everyone can do their own setup.
